docs/reference_notices_hci.adoc

```diff
@@ -24,6 +24,14 @@ include::https://raw.githubusercontent.com/NetAppDocs/common/main/_include/open-
 * link:../media/SolidFire_eSDS_12.5.pdf[Notice for NetApp SolidFire Enterprise SDS 12.5^]
 * link:../media/mNode_12.5.pdf[Notice for Management Node 12.5^]
 * link:../media/ansible-products-notice.pdf[Notice for Ansible Role for Compute Upgrades^]
+* link:../media/Ember_12.3_notice.pdf[Notice for Ember OS 12.3.1^]
```

Comment:
Do we expose "Ember OS"?  We never have in the past.  Should this be "Element"?  Or are we exposing Ember now?  Please double check on this.

Reply:
No that shouldn't be in there - just checked the current page.